css/signin.css

```diff
@@ -7,47 +7,77 @@ box-sizing: border-box;
 }
 
 body {
-margin: 50px auto;
-background-color: rgba(241, 241, 241, 0.9);
+background-color: #fff;
 font-family: 'Poppins', sans-serif;
 font-weight: 700;
+overflow: hidden;
 }
 
-h1 {
-text-align: center;
-color: #1F59BB;
-padding-top: 20px;
+.container {
+display: flex;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/codeclannigeria/CodeClanUI/pullRequest?prid=5689283)